packages/react-celo/src/modals/action.tsx

```diff
@@ -91,6 +91,16 @@ export const ActionModal: React.FC<Props> = ({
   const theme = useTheme();
   const { pendingActionCount, dapp } = useCeloInternal();
 
+  useEffect(() => {
+    if (pendingActionCount > 0) {
+      document.body.style.overflow = 'hidden';
+      document.body.style.paddingRight = '15px';
```

Comment:
why the padding right?

Reply:
from this https://stackoverflow.com/a/64210755 - thoughts?